scripts/minikube-minishift-setup-env.sh

```diff
@@ -81,6 +138,13 @@ case ${1} in
 
         # Create Operators for Operator tests
         sh $SETUP_OPERATORS
+
+        # Create a developer user if it is not created already and change the context to use it after the setup is done
+        kubectl config get-contexts developer-minikube
+        if [ $? -ne 0 ]; then
+          setup_minikube_developer
+        fi
+        kubectl config use-context developer-minikube
```

Comment:
What is it confirming `developer` user already exits?

Reply:
Yeah, in case there is a running minikube cluster, we check if the context is created, if not, we create the context, user, and everything else, and then use the context.